bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/install/IDFDownloadPage.java

```diff
@@ -397,9 +397,10 @@ private void validate()
 			}
 			
 			String idfPyPath = idfPath + File.separator + "tools" + File.separator + "idf.py"; //$NON-NLS-1$ //$NON-NLS-2$
-			if (!new File (idfPyPath).exists())
+			String idfToolsPyPth = idfPath + File.separator + "tools" + File.separator + "idf_tools.py"; //$NON-NLS-1$ //$NON-NLS-2$
+			if (!new File (idfPyPath).exists() && !new File (idfToolsPyPth).exists())
 			{
```

Comment:
Use constants for idf.py and idf_tools.py from the IDFConstants file.

Reply:
Done